css/ampdoc.css

```diff
@@ -351,17 +351,4 @@ amp-autocomplete > textarea,
  */
 [amp-fx^="fly-in"] {
   visibility: hidden;
-}
-
-/**
- * amp-script[nodom] is an element with no UI, similar to amp-analytics.
- */
-amp-script[nodom] {
```

Comment:
This is generally a good thing. But what does `nodom` do otherwise? It might take space accidentally in the style tree just because it's still an element. Do we require `layout=nodisplay` on such elements? If not, here's another idea:

1. We make `amp-script[nodom] {display: none !important}`.
2. We implement `deferredMount(element) => !element.hasAttribute('nodom')`. I.e. `amp-script[nodom]` would be immediate-mount.

Reply:
Oh. This is a much better idea. Thank you